src/main/java/com/codejoust/main/dto/game/GameMapper.java

```diff
@@ -69,7 +69,7 @@ public static Game fromRoom(Room room) {
         for (User user : room.getUsers()) {
             Player player = PlayerMapper.playerFromUser(user);
             player.setColor(colorList.get(index));
-            player.setSolved(new Boolean[room.getNumProblems()]);
+            player.setSolved(new boolean[room.getNumProblems()]);
```

Comment:
What does this line do?

Reply:
This line sets every player to not having solved any of the problems.